src/pykeen/models/base.py

```diff
@@ -199,9 +206,11 @@ def score_r(self, ht_batch: torch.LongTensor, slice_size: Optional[int] = None)
         :param slice_size: >0
             The divisor for the scoring function when using slicing.
 
-        :return: shape: (batch_size, num_relations), dtype: float
+        :return: shape: (batch_size, num_real_relations), dtype: float
             For each h-t pair, the scores for all possible relations.
         """
+        # TODO: this currently compute (batch_size, num_relations) instead,
+        # i.e., scores for normal and inverse relations
```

Comment:
This is an interesting question. I think this should only compute scores for "real" relations, but not for inverses.

In https://github.com/pykeen/pykeen/pull/728 this can be extended to allow to compute relations scores using *only* inverse relations.

Reply:
Can you elaborate a little?